include/qemu/osdep.h

```diff
@@ -85,17 +85,17 @@ extern int daemon(int, int);
 #endif
 #endif
 
+/* enable C99/POSIX format strings (needs mingw32-runtime 3.15 or later) */
+#ifdef __MINGW32__
+#define __USE_MINGW_ANSI_STDIO 1
+#endif
+
```

Comment:
Why was this code-block moved?

Reply:
Noted in [commit message](https://github.com/xqemu/xqemu/pull/266/commits/25302ddd2bd8a392a0000605db67e4deeb19845d):
> This patch moves the definition of __USE_MINGW_ANSI_STDIO to precede
inclusion of any headers. This is done to prevent stdio.h from assigning
a default value which was previously conflicting with our unconditional
definition, resulting in numerous compiler warnings.